metaflow/plugins/kfp/kfp.py

```diff
@@ -9,6 +9,7 @@
 import kfp
 from kfp import dsl
 from kfp.dsl import ContainerOp
+from kfp.dsl._pipeline import PipelineConf
```

Comment:
You should be able to import that from `kfp.dsl`, it's typically bad practice to import from specific modules if you can avoid it (especially if the module is prefixed with `_` as in python that denotes private by convention). This way the `kfp` maintainers can rework the internal structure of the `kfp.dsl` module and not need to update their interfaces (ie. rename `_pipeline.py` to something else).

Reply:
Got it, thanks for the review! This has been updated.